File: CapaPresentacion/Controllers/HomeController.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Web;
using System.Web.Mvc;

namespace CapaPresentacion.Controllers
{
    public class HomeController : Controller
    {
        public ActionResult Index()
        {
            return View();
        }

        public ActionResult About()
        {
            ViewBag.Message = "Your application description page.";

            return View();
        }

        public ActionResult ABMBodegas()
        {
            ViewBag.Message = "ABMBodegas";

            return View();
        }

        public ActionResult ABMCategorias()
        {
            ViewBag.Message = "ABMCategorias";

            return View();
        }
        public ActionResult ABMProductos()
        {
            ViewBag.Message = "ABMProductos";

            return View();
        }

        public ActionResult Bodegas()
        {
            ViewBag.Message = "Bodegas";

            return View();
        }

        public ActionResult Categorias()
        {
            ViewBag.Message = "Categorias";

            return View();
        }
        public ActionResult Productos()
        {
            ViewBag.Message = "Productos";

            return View();
        }

        public ActionResult Contact()
        {
            ViewBag.Message = "Your contact page.";

            return View();
        }
    }
}

File: EvaluacionTecnicaLexica/Controllers/HomeController.vb
Public Class HomeController
    Inherits System.Web.Mvc.Controller

    Function Index() As ActionResult
        Return View()
    End Function

    Function About() As ActionResult
        ViewData("Message") = "Your application description page."

        Return View()
    End Function

    Function Categorias() As ActionResult
        ViewData("Message") = "Categorias de productos"

        Return View()
    End Function

    Function ABMCategorias() As ActionResult
        ViewData("Message") = "ABM de Categorias"

        Return View()
    End Function
    Function ABMProductos() As ActionResult
        ViewData("Message") = "ABM de Productos"

        Return View()
    End Function
    Function Productos() As ActionResult
        ViewData("Message") = "Listado de Productos"

        Return View()
    End Function
    Function ABMBodegas() As ActionResult
        ViewData("Message") = "AMB de Bodegas"

        Return View()
    End Function
    Function Bodegas() As ActionResult
        ViewData("Message") = "Bodegas"

        Return View()
    End Function
    Function Contact() As ActionResult
        ViewData("Message") = "Your contact page."

        Return View()
    End Function
End Class
